File: problemas/1000/1066.dart
import 'dart:io';

void main() {
  List numeros = new List();

  for (int i = 0; i < 5; i++) {
    numeros.add(int.parse(stdin.readLineSync()));
  }

  stdout.writeln('${numeros.where((x) => x % 2 == 0).length} valor(es) par(es)');
  stdout.writeln('${numeros.where((x) => x % 2 != 0).length} valor(es) impar(es)');
  stdout.writeln('${numeros.where((x) => x > 0).length} valor(es) positivo(s)');
  stdout.writeln('${numeros.where((x) => x < 0).length} valor(es) negativo(s)');
}

File: problemas/1000/1066.py
numeros = list()

for i in range(5):
    numeros.append(int(input()))

print(f'{len(list(filter(lambda x: x % 2 == 0, numeros)))} valor(es) par(es)')
print(f'{len(list(filter(lambda x: x % 2 != 0, numeros)))} valor(es) impar(es)')
print(f'{len(list(filter(lambda x: x > 0, numeros)))} valor(es) positivo(s)')
print(f'{len(list(filter(lambda x: x < 0, numeros)))} valor(es) negativo(s)')
